src/parser/transformer.cpp

```diff
@@ -913,8 +914,18 @@ unique_ptr<Statement> Transformer::transformDropTable(CypherParser::KU_DropTable
 }
 
 unique_ptr<Statement> Transformer::transformAlterTable(CypherParser::KU_AlterTableContext& ctx) {
-    return make_unique<DropProperty>(transformSchemaName(*ctx.oC_SchemaName()),
-        transformPropertyKeyName(*ctx.oC_PropertyKeyName()));
+    if (ctx.kU_AlterOptions()->kU_DropColumn()) {
```

Comment:
I would add transformDropProperty(cox) and transformAddProperty(ctx).

Reply:
added these two functions